### PR TITLE
Button stories changed to new structure

### PR DESCRIPTION
**Pull request by shabtaisharon**
#### What does this PR do?
Updates the stories in Button/stories to use the new structure.
#### Where should the reviewer start?
`Button.stories.js`
#### What testing has been done on this PR?
yarn storybook
#### How should this be manually tested?
Go to storybook PR-deployment, and verify that file structure is applied.
#### Any background context you want to provide?
No
#### What are the relevant issues?
#4651
#### Screenshots (if appropriate)
image
#### Do the grommet docs need to be updated?
No
#### Should this PR be mentioned in the release notes?
No
#### Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

File: src/js/components/Button/stories/RoutedButton.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 
 import { Box, Grommet, RoutedButton, Text } from 'grommet';
 import { grommet } from 'grommet/themes';
 
-const RouteButton = () => (
+export const RouteButton = () => (
```
Comment (jcfilben) [inline src/js/components/Button/stories/RoutedButton.js:6]:
```suggestion
export const RoutedButton = () => (
```

File: src/js/components/Button/stories/RoutedButton.js [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
 
-const RouteButton = () => (
+export const RoutedButton = () => (
   <Grommet theme={grommet}>
     <Box align="center" pad="large">
       <Text margin="medium" size="small">
         Note: RoutedButton will soon be deprecated
       </Text>
-      <RoutedButton label="Go" path="/" />
+      <GrommetRoutedButton label="Go" path="/" />
```
Comment (jcfilben) [inline src/js/components/Button/stories/RoutedButton.js:17]: Why was this change made?

Comment (shabtaisharon) [reply src/js/components/Button/stories/RoutedButton.js:17]: i needed to rename the imported `RoutedButton` from grommet in line 6

**Comment by ShimiSun** [comment]
Thank you for your contribution!

**Pull request merged**